clang/test/CXX/drs/cwg5xx.cpp

```diff
@@ -1230,11 +1230,11 @@ namespace cwg598 { // cwg598: yes
   int &t = h(N::i);
 }
 
-namespace cwg599 { // cwg599: partial
+namespace cwg599 { // cwg599: 19
   typedef int Fn();
   struct S { operator void*(); };
   struct T { operator Fn*(); };
-  struct U { operator int*(); operator void*(); }; // #cwg599-U
+  struct U { operator int*(); operator void*(); };
   struct V { operator int*(); operator Fn*(); };
   void f(void *p, void (*q)(), S s, T t, U u, V v) {
```

Comment:
Can you add a pointer to reference case?

Reply:
I've added that in another file along with other contextual implicit conversions with references

Reply:
Thank you, but we still need it here for completeness of the DR test. It's not an issue when a DR test overlap with some other test outside of DR test suite.

Reply:
Sorry, you added reference to pointer cases, which made me realize that pointers to references I've been asking for are prohibited by http://eel.is/c++draft/dcl.ref#5.sentence-1